doc/howto/migrate_instances.md

```diff
@@ -0,0 +1,44 @@
+(migrate-instances)=
+# How to migrate existing LXD instances between servers
+
+To move an instance from one LXD server to another, use the `lxc move` command:
+
+    lxc move [<source_remote>:]<source_instance_name> <target_remote>:[<target_instance_name>]
+
+```{note}
+When moving a container, you must stop it first.
+See {ref}`live-migration` for more information.
+```
+
+You don't need to specify the source remote if it is your default remote, and you can leave out the target instance name if you want to use the same instance name.
+If you want to move the instance to a specific cluster member, specify it with the `--target` flag.
+
+You can add the `--mode` flag to choose a transfer mode, depending on your network setup:
+
```

Comment:
I think it would be beneficial to re-word this in terms of connections opened, as this is its purpose, to work around restrictions on network connection direction.

`pull` - Instruct the target server to connect to the source server.
`push` - Instruct the source server to connect to the target server.
`relay` - Instruct the client to connect to both the source and target servers and transfer data through the client.

Reply:
Also on `relay` the way its worded at the moment implies that the instance is fully downloaded to the client and then re-uploaded, but in reality its not a two-step process, but instead the individual bytes are relayed and do not hit the client's disk.